### PR TITLE
Implementing enemy list

### PR DESCRIPTION
**Pull request by Tischel**
Please test if you can

image

example:
image

### CODE REVIEW

File: DelvUI/Interface/EnemyList/EnemyListConfig.cs [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+    [DisableParentSettings("Position", "Anchor", "HideWhenInactive")]
+    [Section("Enemy List", true)]
+    [SubSection("Health Bar", 0)]
+    public class EnemyListHealthBarConfig : BarConfig
+    {
+        [NestedConfig("Colors", 70)]
+        public EnemyListHealthBarColorsConfig Colors = new EnemyListHealthBarColorsConfig();
+
+        [NestedConfig("Change Alpha Based on Range", 80)]
+        public EnemyListRangeConfig RangeConfig = new EnemyListRangeConfig();
```
Comment (vond82) [inline DelvUI/Interface/EnemyList/EnemyListConfig.cs:46]: never used?

Comment (Tischel) [reply DelvUI/Interface/EnemyList/EnemyListConfig.cs:46]: oopsie

**Review by vond82 — APPROVED** [review]

**Comment by Zeffuro** [comment]
So far so good by testing roulettes.

**Comment by vond82** [comment]
Looks good to merge for me but I'll let you decide since you've been pushing stuff still :)

**Comment by Tischel** [comment]
Ive been leveling WAR while testing this so I found a bunch of stuff
I think its good to merge now

**Comment by Zeffuro** [comment]
No complaints from my side, been very stable for me!

**Pull request merged**